Trimble.Modus.Components/Controls/Toast/TMToastContents.xaml.cs

```diff
@@ -0,0 +1,160 @@
+using Microsoft.Maui.Platform;
+using Trimble.Modus.Components.Enums;
+using Trimble.Modus.Components.Popup.Services;
+
+namespace Trimble.Modus.Components.Controls.Toast;
+
+public partial class TMToastContents : Popup.Pages.PopupPage
```

Comment:
Should this class be `public`? As far as I understand, this should not be exposed to the user

Reply:
Yes it is necessary that the TMToastContentsPage is public because it throws and error stating **"an inconsistency in the accessibility modifiers of your classes"** hence to maintain consistency **both pages are public but contents inside are internal** and now the TMToastContentsPage is not exposed to the user now, as I have changed the accessibility modifier of the constructor from public to internal